db/models/scores_justifications.py

```diff
@@ -0,0 +1,80 @@
+import uuid
+from datetime import datetime
+
+from db import db
+
+
+class ScoresJustifications(db.Model):
+    scores_justifications_id = db.Column(
+        "scores_justifications_id",
+        db.Text(),
+        default=lambda: uuid.uuid4(),
+        primary_key=True,
+    )
+    timestamp = db.Column("timestamp", db.Text(), default=str(datetime.now()))
+    assessment_id = db.Column(
+        "assessment_id",
+        db.Text(),
+        ForeignKey=True,
+    )
+    person_id = db.Column(
+        "person_id",
+        db.Text(),
+        ForeignKey=True,
+    )
+    sub_criteria_id = db.Column(
+        "sub_criteria_id",
+        db.Text(),
+        ForeignKey=True,
+    )
+    score = db.Column(
+        db.Integer(),
+    )
+    justification = db.Column(
+        db.Text(),
+    )
+
+    def __repr__(self):
+        return f"<Sub-Criteria {self.sub_criteria_title}, {self.sub_criteria_id} \
+                for Criteria {self.criteria_id} \
+                of Round {self.round_id}>"
```

Comment:
⛏️ Reprs should resemble how the object is created ie `ScoresJustification(score=5,blah=blah)`, but this would make a great `__str__`